src/detectors/ScanningStrategyDetector.spec.ts

```diff
@@ -66,7 +67,8 @@ describe('ScanningStrategyDetector', () => {
           getRemotes: () => [{ name: 'origin', refs: { fetch: repoPath, push: repoPath } }],
         };
       });
-      const container = createTestContainer({ uri: '/local/path' });
+
+      const container = createTestContainer({ uri: '/local/path', auth: 'auth' });
```

Comment:
Shouldn't this throw an error? The repository does not exist or the auth is incorrect.